lib/util/git/config.spec.ts

```diff
@@ -1,9 +1,22 @@
+import { mocked } from '../../../test/util';
+import { GlobalConfig } from '../../config/global';
 import { simpleGitConfig } from './config';
 
+jest.mock('../../config/global');
+const globalConfig = mocked(GlobalConfig);
```

Comment:
Don't mock, simply set correct value in test

Reply:
Used the mock in the new test

Reply:
```suggestion
```
Please don't mock, simply reset on brfore each and set appropriate value on test

Like here:
https://github.com/renovatebot/renovate/blob/443bef5c52cbf9e706fbd5d65f271ad4c799ae4c/lib/modules/datasource/npm/index.spec.ts#L16-L18
https://github.com/renovatebot/renovate/blob/443bef5c52cbf9e706fbd5d65f271ad4c799ae4c/lib/modules/datasource/npm/index.spec.ts#L339-L345